### ISSUE TITLE
Desktop has disappeared

### ISSUE THREAD

**Issue by Adam-Shaw** [osxfuse/osxfuse#331]
So I'm not that experienced with this sort of thing, but I was trying to set up sshfs on my mac (El Capitan). I accidentally mounted onto my desktop, i.e. I tried to do "sshfs username@server:/transfer /Users/Ansible/Desktop", and hit enter to early before I had completed the file location. I then received the error 
"
mount_osxfuse: mount point /Users/Ansible/Desktop is itself on a OSXFUSE volume
fuse: failed to mount file system: Invalid argument
"
Long story short, now my Desktop folder has disappeared from the file system. Is there anything I can do to recover? I can provide more information as necessary. Thanks

**Comment by bfleischer** [comment]
You simply need to unmount the sshfs volume. Try running 

```
umount  /Users/Ansible/Desktop
```

in Terminal. You can get a list of all mounts by running 

```
mount
```

in Terminal.

**Comment by Adam-Shaw** [comment]
No luck. Got the following response:
[~]$ umount /Users/Ansible/Desktop
umount(/Users/Ansible/Desktop): Resource busy -- try 'diskutil unmount'

Mount shows:
username@server:/transfer on /Users/Ansible/Desktop (osxfuse, nodev, nosuid, synchronous, mounted by Ansible)

**Comment by Adam-Shaw** [comment]
Is it safe to try diskutil unmount? I know it may seem like a basic question, I just don't want to screw something up even more.

**Comment by bfleischer** [comment]
Yes, its safe to run

```
diskutil unmount /Users/Ansible/Desktop
```

Don't worry, your data is not lost. The sshfs mount hides the contents of your Desktop, that's all. If diskutil does not work try rebooting your Mac.

**Comment by Adam-Shaw** [comment]
Diskutil returned:
"Unmount failed for /Users/Ansible/Desktop"
I'll try rebooting and see what happens. Thanks so much for your quick responses!

**Comment by Adam-Shaw** [comment]
Rebooting worked. Thanks so much!

**Comment by bfleischer** [comment]
You are welcome.

**Comment by frankanneveld** [comment]
Without reboot you can do :

`diskutil unmount force /Users/Ansible/Desktop`
